### PR TITLE
kexec: Add append option

### PR DESCRIPTION
**Pull request by rjoleary**
Signed-off-by: Ryan O'Leary <ryanoleary@google.com>

### CODE REVIEW

File: cmds/kexec/kexec_linux.go [hunk 1]
```diff
@@ -28,15 +28,17 @@ import (
...
+	initramfs     string
+	load          bool
+	exec          bool
 }
 
 func registerFlags() *options {
 	o := &options{}
+	flag.StringVarP(&o.appendCmdline, "append", "a", "", "Append to kernel command line")
 	flag.StringVarP(&o.cmdline, "cmdline", "c", "", "Set the kernel command line")
 	flag.BoolVar(&o.reuseCmdline, "reuse-cmdline", false, "Use the kernel command line from running system")
```
Comment (rminnich) [inline cmds/kexec/kexec_linux.go]: Thinking about this ...
The rush $variable was based on a plan 9 model, where $variable meant read /env/variable. 
$/proc/cmdline meant read /proc/cmdline
I just tested this and it works:
% echo $/proc/mounts
sysfs /sys sysfs rw,nosuid,nodev,noexec,relatime 0 0

elvish and other shells doubtless have a way to plug file contents into the cmdline. 
Given that ...
don't we just need ONE cmdline command? 
kexec with no args does nothing for cmdline
kexec -c $/proc/cmdline
in rush will use the cmdline; in bash it is
read i < /proc/cmdline
kexec -c "$i extra kernel args"
in other words, do we need 3 cmdline args where one might do?
for an empty command line
kexec -c ""
for reusing the kernel cmdline
kexec
for changing the kernel cmdline
read c < /proc/cmdline
kexec -c "$c extra args"

i'm just a bit concerned about 3 switches.

Comment (rjoleary) [reply cmds/kexec/kexec_linux.go]: This was based off of kexec-utils manpage: https://www.systutorials.com/docs/linux/man/8-kexec/

I agree, we can do it all with one argument. Why does does kexec-utils use three?

**Comment by rjoleary** [comment]
We no longer want this option and will concat the cmdline from the shell.
